src/Header.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { db, firebase } from './Firebase.utils.js'
+import "./Header.css"
+import Todo from './Todo.js'
+import { TbLoader } from "react-icons/tb";
+
+const Header = () => {
+
+    const [inputVal, setInputVal] = useState("")
+    const [showImage, setShowImage] = useState(false)
+
+
+    // const addTodo = (e) => {
+    //     e.preventDefault()
+    //     // const myCollection =
+    //     db.collection("todos").add({
+    //         todo: inputVal,
+    //         timestamp: firebase.firestore.FieldValue.serverTimestamp()
+    //     })
+    //     setInputVal("")
+    // }
+
+    const addTodo = (e) => {
+        const inputRegex = /^\s*$/
+
+        if (inputRegex.test(inputVal)) {
+            document.querySelector('.enter-todo-input').classList.add('error')
+            alert('Please enter something!')
+            return
+        }
+
+        document.querySelector('.enter-todo-input').classList.remove('error')
```

Comment:
why custom javascript code? React is much more powerful to do this job. Read and Implement how you can change classes in React @Kunal-inno